detox/test/e2e/03.actions.test.js

```diff
@@ -32,34 +32,46 @@ describe('Actions', () => {
   });
 
   it('should type in an element', async () => {
-    let typedText = device.getPlatform() === 'ios' ? 'Type Working 123 אֱבּג абв!!!' : "Type Working!!!";
-    await element(by.id('UniqueId937')).tap();
+    const typedText = device.getPlatform() === 'ios' ? 'Type Working 123 אֱבּג абв!!!' : "Type Working!!!";
+
+    if (device.getPlatform() === 'ios') {
```

Comment:
@LeoNatan could you get around to fixing this, soon?

Reply:
Is there a specific problem here or what we discussed in Slack?

Reply:
Yes, as I explained on Slack

Reply:
Yes, yes, I thought the comment was about the line above, but it's about the .tap(). Looking into it.

Reply:
Cool, let me know what you come up with - this is interesting :)

Reply:
Found another issue with `clearText()`. Fixing.